crates/ika-move-packages/packages/ika_system/sources/dwallet/dwallet_2pc_mpc_coordinator_inner.move

```diff
@@ -1087,25 +1149,11 @@ public(package) fun register_encryption_key(
 /// - **`message`**: The message that has been approved.
 public struct MessageApproval has store, drop {
     dwallet_id: ID,
+    signature_algorithm: u8,
```

Comment:
I know that currently there is only one curve per dWallet, but do we want to specify curve here anyways? Do we want to offer ourselves the option to support more than one curve per dWallet, or not? Maybe better not to, but we can think of what it means